### ISSUE TITLE
Incorrect plotting in Audition

### ISSUE THREAD

**Issue by nanounanue** [dssg/triage#648]
In the following plot (from `dirtyduck`) note that the *best case* doesn't match the higher line in **2016**.

![metric_over_time_precision@10_pct](https://user-images.githubusercontent.com/494528/54849917-9e49bb80-4cb3-11e9-89bf-c82728d17407.png)

If you look more carefully, you will note that some lines are broken (i.e. just stop at 2016 or start at 2016).

The table `distance_table` (lazy naming, BTW, and bad name space) in the `public` schema is complete and doesn't show those jumps:

```sql
select model_group_id, array_agg(raw_value order by train_end_time) from distance_table  group by 1
```
results on:
```
 model_group_id |                        array_agg                        
----------------+---------------------------------------------------------
              1 | {0.294902912621359,0.241676942046856,0.250915750915751}
              2 | {0.294902912621359,0.241676942046856,0.250915750915751}
              3 | {0.294902912621359,0.241676942046856,0.250915750915751}
              4 | {0.445464982778416,0.391737891737892,0.342359767891683}
              5 | {0.445464982778416,0.391737891737892,0.342359767891683}
              6 | {0.336619718309859,0.324528301886792,0.347248576850095}
              7 | {0.346879535558781,0.285714285714286,0.346228239845261}
              8 | {0.343342036553525,0.328398384925976,0.273622047244095}
              9 | {0.346701164294955,0.326328800988875,0.284600389863548}
             10 | {0.429569266589057,0.369369369369369,0.45136186770428}
             11 | {0.452747252747253,0.403147699757869,0.47047619047619}
             12 | {0.44131455399061,0.386651323360184,0.461977186311787}
             13 | {0.456200227531286,0.383449883449883,0.468926553672316}
             14 | {0.341856060606061,0.309976247030879,0.285261489698891}
             15 | {0.341856060606061,0.309976247030879,0.282747603833866}
             16 | {0.341856060606061,0.309976247030879,0.296825396825397}
             17 | {0.340930674264008,0.309976247030879,0.295384615384615}
             18 | {0.358669833729216,0.330213903743315,0.38877755511022}
             19 | {0.367664670658683,0.325581395348837,0.37979797979798}
             20 | {0.390078917700113,0.361179361179361,0.394927536231884}
             21 | {0.393764434180139,0.350502512562814,0.412844036697248}
             22 | {0.388888888888889,0.324848484848485,0.325471698113208}
             23 | {0.391152502910361,0.33125,0.361736334405145}
             24 | {0.374566473988439,0.309776207302709,0.398347107438017}
             25 | {0.405204460966543,0.324549237170596,0.383720930232558}
             26 | {0.393839383938394,0.334061135371179,0.351260504201681}
             27 | {0.373589164785553,0.330347144456887,0.35191637630662}
             28 | {0.405191873589165,0.327453142227122,0.36744966442953}
             29 | {0.416107382550336,0.318906605922551,0.384105960264901}
             30 | {0.445913461538462,0.380281690140845,0.441233140655106}
             31 | {0.448036951501155,0.385050962627407,0.432780847145488}
             32 | {0.440094899169632,0.394332939787485,0.483208955223881}
             33 | {0.447247706422018,0.392773892773893,0.472589792060492}
             34 | {0.390986601705238,0.341957255343082,0.347602739726027}
             35 | {0.422360248447205,0.329545454545455,0.35738255033557}
             36 | {0.411192214111922,0.340522133938706,0.364951768488746}
             37 | {0.419847328244275,0.331026528258362,0.391071428571429}
             38 | {0.443686006825939,0.389461626575029,0.453307392996109}
             39 | {0.443287037037037,0.406799531066823,0.46551724137931}
             40 | {0.442906574394464,0.38735632183908,0.453154875717017}
             41 | {0.455654101995565,0.390728476821192,0.483747609942639}
             42 | {0.43436754176611,0.374262101534829,0.445848375451264}
             43 | {0.435868331441544,0.384167636786962,0.459312839059675}
             44 | {0.433255269320843,0.37410071942446,0.463587921847247}
             45 | {0.450958286358512,0.383734249713631,0.483516483516484}
(45 rows)
```

**Comment by nanounanue** [comment]
*NOTE*: I just realize (sorry) that the name and schema are controlled from the audition configuration file :/

**Comment by nanounanue** [comment]
Closed in #650
